### PR TITLE
Added cohen-sutherland line clipping algorithm

### PR DESCRIPTION
**Pull request by RajPathare**

### CODE REVIEW

File: code/computational_geometry/cohen_sutherland_lineclip/lineclip.c [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    switch(v)
+    {
+    case 0: drawwindow();
+            delay(500);
+            drawline(p1,p2);
+            break;
+    case 1:    drawwindow();
+            delay(500);
+            break;
+    case 2:    p3=resetendpt(p1,p2);
```
Comment (hwchiu) [inline code/computational_geometry/cohen_sutherland_lineclip/lineclip.c]: could you check the indent of all switch cases?

File: code/computational_geometry/cohen_sutherland_lineclip/lineclip.c [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    v=visibility(p1,p2);
+    delay(500);
+    
+    switch(v)
+    {
+    case 0: drawwindow();
+            delay(500);
+            drawline(p1,p2);
+            break;
+    case 1:    drawwindow();
```
Comment (hwchiu) [inline code/computational_geometry/cohen_sutherland_lineclip/lineclip.c]: same as below.

**Comment by RajPathare** [comment]
Sure

**Comment by AdiChat** [comment]
Looks good 👍 

Hope you are enjoying your journey with **OpenGenus**

**Pull request merged**